thunder/tests/test_examine_memory.py

```diff
@@ -98,6 +98,14 @@ def bar2(a, b):  # [5,2], [2,2]
 
 @requiresCUDA
 def test_nanogpt_block():
+    from thunder.tests.test_sdpaex_executor import device_version_support, CudaVersion
+
+    torch_device = torch.device("cuda")
+    # The estimated memory usage is not the same as actual peak memory usage on Hopper
+    if device_version_support(torch_device, CudaVersion(9, 0), CudaVersion(9, 0)):
```

Comment:
Would we want to run the test on device with compute capability more than 9.0? If not then following should do the trick

```suggestion
    if torch.cuda.get_device_capability() >= (9, 0):
```